### ISSUE TITLE
Bug:滚动到页面底部时，返回顶部按钮无限跳动

### ISSUE THREAD

**Issue by ishowman** [viosey/hexo-theme-material#433]
**I certify that I have first consulted** (check all with "x")

**我确认我已经查看了** (使用 "x" 选择)

- [ x ] [Hexo documentation | Hexo 文档](https://hexo.io/docs/)
- [ x ] [Material theme documentation | Material 主题文档](https://material.viosey.com/)
- [ x ] [Material theme issues | Material 主题的其它 issue](https://github.com/viosey/hexo-theme-material/issues?utf8=%E2%9C%93&q=is%3Aissue)

----

**I'm submitting a**  (check one with "x")

**我正在申请**  (使用 "x" 选择)

- [ x ] bug report | 反馈一个 bug
- [ ] feature request | 申请添加新的特性或功能
- [ ] support request | 请求技术支持

- PC chrome浏览器，win7 x64系统
- 当滚动到页面底部时，返回顶部按钮会出现自动添加/删除.btt-visible的类名，导致按钮一直左右跳动，并会出现横向滚动条
![bug-min](https://user-images.githubusercontent.com/16897283/28195350-3a5af70a-687c-11e7-8e6d-f0b00f48c2aa.PNG)

**Comment by cubesky** [comment]
duplicate #404 
fixed in latest `Canery`

**Comment by ghostsf** [comment]
**/layout/_partial/config_css.ejs**

```
<!-- Config CSS -->

<!-- Other Styles -->
<style>
  body, html {
    font-family: <%- theme.fonts.family %>;
**+    overflow-x: hidden;**
  }
```

**Comment by neoFelhz** [comment]
fixed in version [1.5.0](https://github.com/viosey/hexo-theme-material/releases/tag/1.5.0)
